pages/update-profile.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="../css/main.css" />
+    <title>Update profile</title>
+  </head>
+
+  <body>
+    <main>
+      <div class="container">
```

Comment:
Hey Pacy,
Can you avoid using a global class name like container, it may affect other pages when your code is merged...
Can you choose specific class names like profile-container and apply your CSS to it?

Reply:
100% agree!